crates/core/tedge_actors/src/servers/message_boxes.rs

```diff
@@ -177,7 +180,9 @@ mod tests {
     #[tokio::test]
     async fn only_processes_messages_up_to_max_concurrency() {
         let mut builder = ServerMessageBoxBuilder::new("ConcurrentServerMessageBoxTest", 16);
-        let mut test_box = builder.new_client_box();
+        let mut test_box = SimpleMessageBoxBuilder::new("Test box", 16);
+        test_box.connect_sink(NoConfig, &builder.connect_client(test_box.get_sender()));
+        let mut test_box = test_box.build();
```

Comment:
Why not the former variant itself, which is the same but simpler?
```suggestion
    let mut test_box = builder.new_client_box();
```

Reply:
This is a residue of my attempt to fully remove the `new_client_box()` test helper.

Reverted.